docs/cubeviz/import_data.rst

```diff
@@ -43,3 +43,24 @@ method, which takes as input either the name of a local file or a
     cubeviz.load_data("/Users/demouser/data/cube_file.fits")  # doctest: +SKIP
     cubeviz.app
 
+
+If instead you would like to load data you have created within the notebook,
+you can do the following::
+
+    import numpy as np
+    from astropy import units as u
+    from astropy.wcs import WCS
+    from specutils import Spectrum1D
+
+    flux = np.arange(16).reshape((2, 2, 4)) * u.Jy
+    wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
```

Comment:
Just a reminder, the WCS axis order is the opposite of numpy, so if you wanted your length 4 axis to be the wavelength then `CTYPE1` should be your `WAVE-LOG`.

Reply:
This is straight from `jdaviz/conftest.py`, does that mean it is wrong there as well?

Reply:
Got it.